diffkemp/simpll/ModuleAnalysis.h

```diff
@@ -1,4 +1,4 @@
-//===------------- Transforms.h - Simplifications of modules --------------===//
+//===----------- ModuleAnalysis.h - Simplifications of modules ------------===//
```

Comment:
Could you also change the header comment for this file (and the .cpp file) so that it explains its current purpose better?

Reply:
That's a good idea - the current header omits the comparison part, which has been a part of the file for a long time.

Reply:
I meant the entire file commentary. It now says that the file contains declarations of functions that do simplification, but it should be something like:
"This file contains the declaration of functions that do the actual semantic comparison of functions (modules)."

Analogously for ModuleAnalysis.cpp